comixed-frontend/src/app/library/adaptors/library.adaptor.spec.ts

```diff
@@ -312,12 +315,16 @@ describe('LibraryAdaptor', () => {
 
   describe('consolidating the library', () => {
```

Comment:
I went back and forth on it. Ultimately it's still consolidation since you don't have to move the comics to a new directory to use this feature: it's going to be the way you convert comics, delete comics marked for removal, etc. as well as moving them.

Reply:
ok i understand.